src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -30,13 +30,9 @@
  *
  * @author Oliver Burn
  * @author <a href="mailto:dobratzp@ele.uri.edu">Peter Dobratz</a>
+ * @noinspection ClassWithTooManyDependents
  */
 public final class TokenTypes {
-    // The following three types are never part of an AST,
-    // left here as a reminder so nobody will read them accidentally
-
-    // These are the types that can actually occur in an AST
-    // it makes sense to register Checks for these types
 
```

Comment:
These comments were left here by mistake. The lines to which they were related have been removed earlier.